### ISSUE TITLE
Windows IA32 symbol zip file does not contain symbols

### ISSUE THREAD

**Issue by mbacchi** [brave/brave-browser#891]
Similar to the x86_64 zip file issue recently, the symbols need to be added to the zip file.

```
[mbacchi@host 0.54.3]$ zipinfo -l brave-v0.54.3-win32-ia32-symbols.zip
Archive:  brave-v0.54.3-win32-ia32-symbols.zip
Zip file size: 352699 bytes, number of entries: 2
-rw-rw-rw-  2.0 fat  2043390 b-   346892 defN 18-Aug-29 22:19 LICENSES.chromium.html
-rw-rw-rw-  2.0 fat    17501 b-     5575 defN 18-Aug-29 18:41 LICENSE
2 files, 2060891 bytes uncompressed, 352467 bytes compressed:  82.9%
```

**Comment by mbacchi** [comment]
This looks to be the same cause as the x86_64 [issue](https://github.com/brave/brave-browser/issues/809). I tried running the `dump_syms.exe` against a `.pdb` file and got the following:

```
PS C:\jenkins\win\src\out_x86\Release> .\dump_syms.exe .\brave.exe.pdb
CoCreateInstance CLSID_DiaSource {E6756135-1E65-4D17-8576-610761398C3C} failed (msdia*.dll unregistered?)
Open failed
```

I have found that E6756135-1E65-4D17-8576-610761398C3C refers to the `msdia80.dll` library, which I downloaded from [here](https://github.com/electron/chromium-breakpad/blob/master/msdia80.dll) and tried installing in the current directory, but it doesn't work even if I add the current directory to the path.

I will continue digging into this issue.
